tests/Etsy/EtsyApiBuildRequestTest.php

```diff
@@ -76,7 +76,7 @@ public function testValidData()
 				"tags" => array('fashion, othertag'),
 				"who_made" => "collective",
 				"is_supply" => true,
-				"when_made" => "2010_2015",
+				"when_made" => "2010_2016",
```

Comment:
In order to not have to change this every year, let's change this value to `2000_2009`. What do you think?